airbyte-integrations/connectors/source-mysql/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlSourceConfigurationSpecification.kt

```diff
@@ -321,7 +321,7 @@ data object UserDefinedCursor : CursorMethodConfiguration
 @JsonSchemaDescription(
     "<i>Recommended</i> - " +
         "Incrementally reads new inserts, updates, and deletes using Mysql's <a href=" +
-        "\"https://docs.airbyte.com/integrations/sources/mssql/#change-data-capture-cdc\"" +
+        "\"https://docs.airbyte.com/integrations/sources/mysql/#change-data-capture-cdc\"" +
```

Comment:
good catches! should these changes be in a separate PR? probably, because you'll have to bump the version of source-mysql